lib/src/rules/prefer_equal_for_default_values.dart

```diff
@@ -7,11 +7,13 @@ import 'package:analyzer/dart/ast/token.dart';
 import 'package:analyzer/dart/ast/visitor.dart';
 import 'package:linter/src/analyzer.dart';
 
-const _desc = r'Prefer equal for default values.';
+const _desc = r'Use `=` to separate a named parameter from its default value.';
```

Comment:
I think backticks are not allowed here as the desc appears in output without markdown.

Reply:
I think people will still be able to read it :) but if there's an established style for quotes in linter descs, I should follow it...

Reply:
Hmmm...  We do this elsewhere actually:

https://github.com/dart-lang/linter/blob/36bc8fbf15e0156a8f25d87d60cffd41b412f892/lib/src/rules/prefer_is_not_empty.dart#L18

So I'd just leave the formatting in.

If we want to get serious about unifying the format, we should update the docs.  @a14n : feel free to open an issue if that interests you.

Thanks!